number/src/serialize.rs

```diff
@@ -81,14 +81,24 @@ fn ceil_div(num: usize, div: usize) -> usize {
     (num + div - 1) / div
 }
 
-pub fn write_polys_file<T: FieldElement>(
-    file: &mut impl Write,
-    degree: DegreeType,
-    polys: &[(&str, Vec<T>)],
-) {
+pub fn write_polys_file<T: FieldElement>(file: &mut impl Write, polys: &[(&str, Vec<T>)]) {
```

Comment:
This is also a bit unrelated, but all constants must have the same length anyway, so we can infer that length from the values without passing the degree.

Reply:
Ah, yes I did the same in my PR which was already merged, so there might be some conflicts/diffs here too